src/core/digest.js

```diff
@@ -26,7 +26,9 @@
  */
 import _ from 'lodash';
 import 'webcrypto-shim';
-import { TextEncoder } from 'fastestsmallesttextencoderdecoder';
+import TextEncoderPolyfill from 'fastestsmallesttextencoderdecoder';
+
+const { TextEncoder } = TextEncoderPolyfill;
```

Comment:
why do you need to destruct it ?  `import { TextEncoder } from 'fastestsmallesttextencoderdecoder';` should work no ?

Reply:
It is not working, because the `commonjs` plugin of `rollup` is stupid sometimes and it cannot detect the named exports correctly ( mostly from minified codes ).
@krampstudio 's solution is working, if you add these lines into `rollup.config.js`
```
commonJS({
  namedExports: {
    fastestsmallesttextencoderdecoder: [ 'TextEncoder' ]
  }
}),
```

I prefer to add these lines.

Reply:
The issue resolved by @btamas suggestion. Thank you!